src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'point')
+def point_at_obj(robot, bottle, manip=None):
+    '''
+    @param robot The robot performing the point
+    @param bottle The bottle to point
+    @param manip The manipulator to perform the grasp. if None 
+        the active manipulator on the robot is used
+    '''
+
+    if manip is None:
+        manip_idx = robot.GetActiveManipulatorIndex()
+    else:
+        with manip.GetRobot():
+            manip.SetActive()
+            manip_idx = manip.GetRobot().GetActiveManipulatorIndex()
+		
+    # compute T_ow
+    T0_w_0 = bottle.GetTransform()
+    T0_w_1 = numpy.identity(4)
+
+    # compute T_we. Tuned once and used forever onward 
+    # Insures pointer points at object with right arm
+    arm_start =  numpy.array([[ 0.07277, -0.71135,  0.69905,  0.58575],
```

Comment:
I'm afraid this is super-brittle. I'm guessing this is the relative transform from the hand base frame to the tip of the finger that is pointing. One major issue is that this assumes that the hand is already preshaped to the desired shape, a shape that is not part of this function at all. 
My suggestion is to create an empty link in the URDF and give it a descriptive name like `pointer` and just get its relative transform. It's also quite generalizable. What do you think @mkoval?